src/axolotl/utils/models.py

```diff
@@ -467,6 +492,13 @@ def load_model(
         model_kwargs["quantization_config"] = BitsAndBytesConfig(
             **bnb_config,
         )
+    elif cfg.adapter == "lora" and cfg.load_in_8bit:
+        bnb_config = {
+            "load_in_8bit": True,
+        }
```

Comment:
Previously, we did not have this. What is the effect of this change?

Reply:
There is a warning currently in transformers that passing load_in_8bit will be deprecated soon and that we should use the quantization config instead.